java/src/main/java/com/google/oak/functions/weather_lookup_client/Main.java

```diff
@@ -43,12 +44,12 @@ public static void main(String[] args) throws Exception {
     builder = AttestationClient.addApiKey(builder, EMPTY_API_KEY);
     ManagedChannel channel = builder.build();
 
-    // Attest a gRPC channel.
-    AttestationClientNoninteractive client = new AttestationClientNoninteractive(channel);
+    // Create gRPC client stub.
+    StreamingSessionGrpc.StreamingSessionStub client = StreamingSessionGrpc.newStub(channel);
 
     // Test request coordinates are defined in `oak_functions/lookup_data_generator/src/data.rs`.
     byte[] requestBody = "{\"lat\":0,\"lng\":0}".getBytes(UTF_8);
-    byte[] response = client.send(requestBody);
+    byte[] response = AttestationClientNoninteractive.invoke(client::stream, requestBody);
```

Comment:
Does `::stream` return a `Function<StreamObserver<ResponseWrapper>, StreamObserver<RequestWrapper>>` out of the box?

Reply:
Yes, it's a Java function reference